backend/src/v5/processors/users.js

```diff
@@ -30,9 +30,16 @@ const { publish } = require('../services/eventsManager/eventsManager');
 const { sendEmail } = require('../services/mailer');
 const { templates } = require('../services/mailer/mailer.constants');
 
-Users.signUp = async (newUserData) => {
+Users.signUp = async (newUserData, generatePassword) => {
```

Comment:
I don't think we should have a flag here like this

we could just check the data, if it's sso we know we need to generate password?

Reply:
you mean check if newUserData.password is empty then generate it ? so no flag at all ?